distribution/docs/src/main/resources/_contents/_developing/developing-intro-contents.adoc

```diff
@@ -0,0 +1,816 @@
+|   Operating System   |cEEE     Windows     ||cEEE     Linux       ||cEEEMac OS X      ||cEEE       Solaris          | |
+|                      \---------------------/\---------------------/\------------------/\----------------------------/ |
+|                      /------------------------------------------------------------------------------\/--------------\ |
+|       Hardware       |cEEE                       x86                                                ||cEEE SPARC    | |
+|                      \------------------------------------------------------------------------------/\--------------/ |
++-----------------------------------------------------------------------------------------------------------------------+
+----
+
+As depicted in the architectural diagram above, ${branding} runs on top of an OSGi framework, a Java virtual machine (JVM), several choices of operating systems, and the physical hardware infrastructure.
+The items within the dotted line represent the ${branding} out-of-the-box.
+
+${branding} is a customized and branded distribution of http://karaf.apache.org/[Apache Karaf].
+${branding} could also be considered to be a more lightweight OSGi distribution, as compared to Apache ServiceMix, FUSE ESB, or Talend ESB, all of which are also built upon Apache Karaf.
+Similar to its peers, ${branding} incorporates additional upstream dependencies (https://tools.codice.org/#DDFArchitecture-AdditionalUpstreamDependencies).
+
+${branding} as a framework hosts ${branding} applications, which are extensible by adding components via OSGi.
+The best example of this is the ${branding} Catalog (API), which offers extensibility via several types of Catalog Components.
+The ${branding} Catalog API serves as the foundation for several applications and resides in the applications tier.
+
+The Catalog Components consist of Endpoints, Plugins, Catalog Frameworks, Sources, and Catalog Providers.
+Customized components can be added to ${branding}.
+
+==== Nomenclature
+
+Capability:: A general term used to refer to an ability of the system
+Application:: One or more features that together form a cohesive collection of capabilities
+Component:: Represents a portion of an Application that can be extended
+Bundle:: Java Archives (JARs) with special OSGi manifest entries.
+Feature:: One or more bundles that form an installable unit;  Defined by Apache Karaf but portable to other OSGi containers.
+
+==== OSGi Core
+
+${branding} makes use of OSGi v4.2 to provide several capabilities:
+
+* Has a Microkernel-based foundation, which is lightweight due to its origin in embedded systems.
+* Enables integrators to easily customize components to run on their system.
+* Software applications are deployed as OSGi components, or bundles.
+Bundles are modules that can be deployed into the OSGi container (Eclipse Equinox OSGi Framework by default).
+* Bundles provide flexibility allowing integrators to choose the bundles that meet their mission needs.
```

Comment:
Should this and the following line be sub-list elements under the `Software applications are deployed...` bullet?